zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/EncryptionUtils.java

```diff
@@ -64,4 +68,15 @@ public static InputStream encryptionInputStream(InputStream inputStream, SecretK
     public static InputStream encryptionInputStream(Path path, SecretKey key) throws IOException {
         return encryptionInputStream(Files.newInputStream(path), key);
     }
+
+    public static String createMD5CheckSum(String value) {
+        try {
+            byte[] bytesOfMessage = value.getBytes(StandardCharsets.UTF_8);
+            MessageDigest md = MessageDigest.getInstance("MD5");
+            byte[] theMD5digest = md.digest(bytesOfMessage);
+            return DatatypeConverter.printHexBinary(theMD5digest).toLowerCase();
+        } catch (NoSuchAlgorithmException e) {
```

Comment:
This can only happen when calling `getInstance`, so you should only surround that line with the try/catch.